misc/docs/source/tutorial/code_snippets/stationxml_file_from_scratch.rst

```diff
@@ -0,0 +1,85 @@
+=============================================================================
+Creating a StationXML file from Scratch
+=============================================================================
+
+Creating a custom StationXML file is a task that sometimes comes up in
+seismology. This sections demonstrates how to it with ObsPy. Please note that
+this is not necessarily easier or more obvious then directly editing an XML
+file but it does provider tighter integration with the rest of ObsPy and can
+guarantee a valid result at the end.
+
+Note that this assumes a certain familiarity with the `FDSN StationXML standard
+<http://www.fdsn.org/xml/station/>`_. We'll create a fairly simplistic
```

Comment:
https
(Funnily enought, over https, the font is blocked and it looks _better_ because the font is not so small.)